test/assembly/DockerTest.java

```diff
@@ -40,7 +40,7 @@ public void bootup() throws InterruptedException, TimeoutException, IOException
                 "docker", "run", "--name", "typedb",
                 "--rm", "-t", "-p", String.format("%d:%d", typeDBPort, typeDBPort),
                 "bazel:assemble-docker",
-                "/opt/typedb-all-linux-x86_64/typedb", "server", "--diagnostics.reporting.errors=false", "--diagnostics.reporting.statistics=false"
+                "/opt/typedb-all-linux-x86_64/typedb", "server"
```

Comment:
The server is in release mode only if this test is run with `--//server:config=release`. What if we would want to...